ci

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -ev # Ref https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps
+set -x
+
+case "${1:?}"-"${2:?}" in
+    before_install-*)
+        true
+        ;;
+    install-dialyzer)
+        BuildDir="${3:?}"
+        ( cd "${BuildDir:?}" && ./rebar3 tree; )
+        ( cd "${BuildDir:?}" && ./rebar3 dialyzer -u true -s false; )
+        ;;
+    install-*)
+        true
+        ;;
+    script-test)
+        BuildDir="${3:?}"
+        ( cd "${BuildDir:?}" && make test; )
+        ;;
+    script-dialyzer)
+        BuildDir="${3:?}"
+        ( cd "${BuildDir:?}" && make dialyzer; )
+        ;;
+    script-xref)
+        BuildDir="${3:?}"
+        ( cd "${BuildDir:?}" && ./rebar3 xref; )
+        ;;
+    after_failure-test)
+        BuildDir="${3:?}"
+        ls -l "${BuildDir:?}"/_build/dev3/rel/ae_core/log/*
```

Comment:
I understand the dev3/dev1/dev2 nodes are not generated so I understand this can be deleted.